src/main/java/de/cas/mse/exercise/SumDigPower.java

```diff
@@ -5,37 +5,46 @@
 
 public class SumDigPower {
 
-	public List<Long> sumDigPow(long a, long b) {
+	public List<Long> sumDigPow(long start, long end) {
```

Comment:
Maybe a better name would be something with "find" or "print" better since this is what we are doing